projects/Multipharma/info.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Multipharma",
+  "description": "Met de Multipharma app is het eenvoudig om jouw geneesmiddelen te reserveren in de apotheek van uw keuze en om uw apotheker op afstand om advies te vragen.",
+  "publisher": "In The Pocket",
+  "developers": [],
```

Comment:
You could add a single developer here and leave its Github username empty, but fill in the company. That way In The Pocket is listed as the agency (involved company). I think you might want to [put MultiPharma as the publisher of the app](https://apps.apple.com/be/developer/multipharma/id1476230936)?
```
{
  "githubUserName": "",
  "companyName": "In The Pocket"
}

Reply:
Fixed!